src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -101,6 +101,12 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.custom_command('verify', 'iot_dps_certificate_verify')
         g.custom_command('update', 'iot_dps_certificate_update')
 
+    # iot dps identity commands
+    with self.command_group('iot dps identity', client_factory=iot_service_provisioning_factory, is_preview=True) as g:
+        g.custom_command('assign', 'iot_dps_identity_assign')
+        g.custom_show_command('show', 'iot_dps_identity_show')
+        g.custom_command('remove', 'iot_dps_identity_remove')
```

Comment:
Do we need to consider adding `confirmation=true` for the remove command?

Reply:
There is no need, a command that is similar to it `az iot hub identity remove` does not have the confirmation.